server/api/src/rsRmColl.cpp

```diff
@@ -100,7 +100,7 @@ namespace
         else {
             if ( isTrashPath( rmCollInp->collName ) == False &&
                     getValByKey( &rmCollInp->condInput, FORCE_FLAG_KW ) != NULL ) {
-                rodsLog( LOG_NOTICE,
+                rodsLog( LOG_DEBUG,
```

Comment:
We've left it as NOTICE for a long time since recursive removal is not an everyday practice in archival systems.

However, as iRODS sees more general user usage, this is a much more common occurrence.

I think this is the right thing to do - but perhaps we wait until 4.3?    Anyone have strong opinions?

Reply:
Just let me know and I can drop this commit and/or move it to another PR for discussion. Leaving this one to the community

Reply:
sysadmins really should not be learning data is being deleted from their logs, as that is a normal filesystem activity.

DEBUG is fine.   4.3 is already going to be much more noisy by default, and all in glorious parseable JSON.